website/blog/2018-07-27-removing-babels-stage-presets.md

```diff
@@ -16,7 +16,7 @@ We didn't make this decision lightly and wanted to show the context behind the i
 
 ## Some History
 
-A Babel preset is a shareable list of plugins.
+A Babel preset is a sharable list of plugins.
```

Comment:
Not a huge deal, but `shareable` seems fine?

Reply:
Both were used. I tend to pick the more dominant spelling w/in a repo or the one that would be dominant for a region. In this case, the short form was used 2:1. Notably, the short form is an en-US variant (and from memory this repo feels en-GB), and it looks like it's on the receding side. I'm happy to switch the `sharable`s to `shareable`.

Reply:
Yeah... we're wildly inconsistent there, either way, not a big deal 😄